python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -3029,10 +3027,10 @@ def __init__(self):
         ([2, 7, 4], torch.float32, True),
     ])
     def forward(self, val):
-        return torch.ops.aten.cumsum(val, 1)
+        return torch.ops.aten.cumsum(val, dim=-2)
```

Comment:
drive by comment: In general, existing tests should not be modified. E2E tests are relatively cheap, so if you want to test an edge case, you can just add a new test. Having more coverage of edge cases is always welcomed.

Also, I don't think we need to rename to `AtenCumsum...` the convention so far has been to not include the `Aten` part, although `Aten_EmbeddingBag...` above did slip through the cracks in this regard.

Reply:
Okay I get it. Now I keep the existing tests unchanged and have added a new test.